custom/ewsghana/reminders/first_soh_reminder.py

```diff
@@ -11,7 +11,8 @@ def recipients_filter(self, user):
         if not roles:
             return False
         needs_reminders = string_to_boolean(user.user_data.get('needs_reminders', "False"))
-        return any([role != IN_CHARGE_ROLE for role in user.user_data.get('role', [])]) and needs_reminders
+        return any([role != IN_CHARGE_ROLE for role in user.user_data.get('role', [])]) and \
+            needs_reminders and user.location
```

Comment:
all of these users are `CommCareUser`s, right? if so we should just check `user.location_id` to prevent the cost of doing the location lookup